### PR TITLE
Style each moving platform with a different color

### PR DESCRIPTION
**Pull request by willemarcel**

### CODE REVIEW

**Comment by LanesGood** [comment]
@willemarcel I've pushed the changes I described. ~I couldn't get the expression below to work - addition of this would be a nice to have, if anything jumps out at you for why this didn't correctly render.~

**Edit**: Interpolation worked fine - I just had the values set too low/wide-ranged to notice the differences.

**Comment by LanesGood** [comment]
Unclear why the build is failing, but I think this is ready for review.

**Comment by willemarcel** [comment]
thanks, @LanesGood! I'm going to review it on Monday.

**Comment by LanesGood** [comment]
@willemarcel icons have been added to mapbox style, using the same name from the codebase.

File: src/components/timeline/map.js [hunk 1]
```diff
@@ -114,28 +116,24 @@ export function DeploymentMap({
...
+                  ["exponential", 0.3],
+                  ["zoom"],
+                  0,
+                  0.1,
+                  12,
+                  0.6,
+                  18,
+                  1,
+                  22,
+                  1.5,
```
Comment (LanesGood) [inline src/components/timeline/map.js]:
```suggestion
                  0,
                 0.25,
                  12,
                  0.75,
                  18,
                  1.25,
                  22,
                  1.75
```
@willemarcel proposed change to make icons slightly larger

**Comment by LanesGood** [comment]
Added circle behind icons, and updated icon colors on mapbox studio.

**Review by LanesGood — APPROVED** [review]
Updated icons and backgrounds on mapbox; could use a final style check @willemarcel (currently cached for my machine)
image

**Pull request merged**